packages/js/src/core/fetch-token.ts

```diff
@@ -22,11 +22,11 @@ type TokenSnakeCaseResponse = {
 };
 
 export type RefreshTokenTokenResponse = {
-  scope?: string[];
+  scopes?: string[];
```

Comment:
should be `scope: string` according to the server response

References:
- [CodeTokenResponse](https://www.notion.so/silverhand/CodeTokenResponse-6b7773814102427083fafe7d6e73e3c8)
- [HTTP API /oidc/token](https://www.notion.so/silverhand/oidc-token-4a502a3f986b4004b11aad1faa681458)

Related Merged PR: https://github.com/logto-io/js/pull/141